src/test/unit/extansion.test.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  Env0EnvironmentsProvider,
+  Environment,
+} from "../../env0-environments-provider";
+import * as myExtension from "../../extension";
+import type vscode from "vscode";
+import { getCurrentBranchWithRetry } from "../../utils/git";
+jest.mock("../../utils/git");
+
+const mockGetCurrentBranch = (branchName: string) => {
```

Comment:
the function you mock is `getCurrentBranchWithRetry`, not `GetCurrentBranch`.
Hence, this should be called `mockGetCurrentBranchWithRetry`

Reply:
f5a6db27b1c3fa6bd482c8ecd5b6f0c24bbadc83